lib/stack_master/template_utils.rb

```diff
@@ -6,8 +6,14 @@ module TemplateUtils
     extend self
 
     def identify_template_format(template_body)
-      return :json if template_body =~ /^{/x # ignore leading whitespaces
-      :yaml
+      # if the first non-whitespace character across all lines is a '{'
+      json_pattern = Regexp.new('\A\s*{', Regexp::MULTILINE)
```

Comment:
Could this be stored in a constant?

Reply:
I reflexively went to do that, but wasn't sure if we want to take the performance hit to compile it whenever we include the file. Doubt it's honestly a performance cost worth worrying about, though?

Reply:
Yeah I doubt it would make a difference in the typical use case - `stack_master apply us-east-1 my-stack`.

Reply:
https://github.com/envato/stack_master/pull/335/commits/ec99ebdb2ad9f578f860a30fdcb55dcb91ab8e1c